src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo.png"
```

Comment:
The `alt` attribute for the image should be descriptive. Instead of 'logo.png', use a descriptive text like 'Company Logo' to improve accessibility.